code/modules/projectiles/gun.dm

```diff
@@ -21,7 +21,14 @@
 	item_flags = NEEDS_PERMIT
 	attack_verb_continuous = list("strikes", "hits", "bashes")
 	attack_verb_simple = list("strike", "hit", "bash")
-
+	light_range = 3
+	light_system = OVERLAY_LIGHT
+	light_on = FALSE
+	light_color = LIGHT_COLOR_ORANGE
+	light_power = 0.5
```

Comment:
color will not show up very strongly with this power. You mentioned lowering it, why was that?

Reply:
> color will not show up very strongly with this power. You mentioned lowering it, why was that?

The original version was a large, bright flash, which was horrible to look at. I'll play about with the numbers some more, directional lighting was suggested.